config/configuration.toml.in

```diff
@@ -46,6 +46,12 @@ disable_block_device_use = @DEFDISABLEBLOCK@
 # 
 #enable_debug = true
 
+# Disable the customizations done in the runtime when it detects
+# that it is running on top a VMM. This will result in the runtime
+# behaving as it would when running on bare metal.
+# 
+#disable_nesting_checks = true
```

Comment:
Can you explain in the comments in the file itself:

- why this is not the default?
- what scenarios it is *not* appropriate to enable this for?

Reply:
This should not be the default. This can be made the default once the KVM bug is fixed to allow us to run with this configuration on systems with the latest kernels. For older kernels we should keep this.